website/docs/getting-started/installation.md

```diff
@@ -0,0 +1,201 @@
+---
+id: installation
+title: ChaosCenter Cluster Scope Installation
+sidebar_label: Installation
+---
+
+---
+
+## Prerequisites
+
+Before deploying LitmusChaos, make sure the following items are there
+
+- Kubernetes 1.15 or later
```

Comment:
We are supporting for 1.17 or later

cc: @ksatchit

Reply:
Have updated the same in prerequisites